extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1214,57 +1214,63 @@ public static function getCurrentStagingHost( $dbName='', $default='', $host = n
 	 * @throws \Exception
 	 */
 	static public function getLocalEnvURL( $url, $forcedEnv = null ) {
```

Comment:
Split out the portion of this method that reformatted the server part of the URL so it could be used elsewhere.